Sources/Foundation.swift

```diff
@@ -158,14 +158,14 @@ extension JSON {
         public var nsNoNull: Any? {
             switch self {
             case .null: return nil
-            case .bool(let b): return NSNumber(value: b)
-            case .string(let s): return s
-            case .int64(let i): return NSNumber(value: i)
-            case .double(let d): return d
-            case .decimal(let d): return d
-            case .object(let obj): return obj.nsNoNull
-            case .array(let ary):
-                return ary.flatMap({$0.nsNoNull})
+            case let .bool(b): return NSNumber(value: b)
+            case let .string(s): return s
+            case let .int64(i): return NSNumber(value: i)
+            case let .double(d): return d
+            case let .decimal(d): return d
+            case let .object(obj): return obj.nsNoNull
+            case let .array(ary):
+                return ary.flatMap {$0.nsNoNull}
```

Comment:
Functional methods should have parentheses. Trailing-closure style should only be used for closures executed for side effects (such as `DispatchQueue.async`).

This has actually been published by Erica Sadun as the [Rule Of Kevin](http://ericasadun.com/2015/11/17/a-handful-of-swift-style-rules-swiftlang/) (where I'm the Kevin).

Reply:
> This has actually been published by Erica Sadun as the Rule Of Kevin (where I'm the Kevin).

I read [Swift Style](https://www.amazon.com/Swift-Style-Opinionated-Guide-Language/dp/1680502352/) a few months ago and as soon as I saw this change was hoping you would reference it. Hilarious.